spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -604,6 +604,122 @@ definitions:
+           units: m
+           desc: >
+             The semi major axis of the estimated horizontal error ellipse at
+             the user-configured confidence level; zero implies invalid.
+       - semi_minor:
+           type: float
+           units: m
+           desc: >
+             The semi minor axis of the estimated horizontal error ellipse at
+             the user-configured confidence level; zero implies invalid.
+       - orientation:
+           type: float
+           units: deg
+           desc: >
+             The orientation of semi major axis of the estimated horizontal
+             error ellipse with respect to North.
+
+
+ - MSG_POS_LLH_ACC:
+    id: 0x0218
+    short_desc: Geodetic Position and Accuracy
+    desc: >
+      This position solution message reports the absolute geodetic coordinates
+      and the status (single point vs pseudo-absolute RTK) of the position
+      solution as well as the estimated horizontal, vertical, cross-track and
+      along-track errors.  The position information and Fix Mode flags should
+      follow the MSG_POS_LLH message. Since the covariance matrix is computed in
+      the local-level North, East, Down frame, the estimated error terms follow
+      with that convention.
+
+
+      The estimated errors are reported at a user-configurable confidence level.
+      The user-configured percentile is encoded in the percentile field.
+    fields:
+        - tow:
+            type: u32
+            units: ms
+            desc: GPS Time of Week
+        - lat:
+            type: double
+            units: deg
+            desc: Latitude
+        - lon:
+            type: double
+            units: deg
+            desc: Longitude
+        - height:
+            type: double
+            units: m
+            desc: Height above WGS84 ellipsoid
+        - h_accuracy:
+            type: float
+            units: m
+            desc: Estimated horizontal error at the user-configured confidence level; zero implies invalid.
+        - v_accuracy:
+            type: float
+            units: m
+            desc: Estimated vertical error at the user-configured confidence level; zero implies invalid.
+        - ct_accuracy:
+            type: float
+            units: m
+            desc: Estimated cross-track error at the user-configured confidence level; zero implies invalid.
+        - at_accuracy:
+            type: float
+            units: m
+            desc: Estimated along-track error at the user-configured confidence level; zero implies invalid.
+        - h_ellipse:
+            type: EstimatedHorizontalErrorEllipse
+            desc: The estimated horizontal error ellipse at the user-configured confidence level.
+        - percentile:
+            type: u8
+            desc: Configured percentile for the estimated position error
+            fields:
+              - 0-7:
+                  desc: Percentile
+                  values:
+                    - 0: no scaling
```

Comment:
scaling of what?

Reply:
After discussion with James and Larry, I've removed the `no scaling option`. It should not be needed.